doc/dune

```
@@ -22,17 +22,26 @@
 ;   odoc_for_authors.mld
 ;   parent_child_spec.mld))
 
+; The driver is also used to collect benchmarks. The benchmark result is always
+; generated but is promoted only by the @bench alias.
+
 (rule
- (alias docgen)
- (deps
-  (:x driver.mld)
-  (glob_files *.ml*)
-  (glob_files *.png)
-  (glob_files library_mlds/*.mld)
-  (package odoc))
+ (targets driver.mld.corrected driver-benchmarks.json)
  (enabled_if
   (> %{ocaml_version} 4.11))
+ (deps
+  (glob_files *.mld)
+  (glob_files library_mlds/*)
+  (glob_files examples/*.ml*))
```

`package odoc` is missing!
```suggestion
 (deps
  (package odoc)
  (glob_files *.mld)
  (glob_files library_mlds/*)
  (glob_files examples/*.ml*))
```

---

This perhaps fixed the random failures you mention above ? I have not experienced those since this patch.